accelerator/core/mam_core.c

```diff
@@ -287,15 +274,14 @@ status_t ta_mam_written_msg_to_bundle(const iota_client_service_t *const service
     }
 
     ep_remain_sk_local = 1 << endpoint_depth;
-    while (ep_remain_sk_local) {
+    while (ep_remain_sk_local > 1) {
```

Comment:
Why?

Reply:
We should leave a mss key here for publishing the MAM announcement which announce the next channel ID

Reply:
I supposed `ep_remain_sk_local` is of unsigned integer, right? If so, `while (ep_remain_sk_local)` is the same as `while (ep_remain_sk_local > 1)`.

Reply:
In the updated one, the condition doesn't include the situation that `ep_remain_sk_local == 1`